src/vuestic-theme/vuestic-components/vuestic-file-upload/VuesticFileUploadGalleryItem.vue

```diff
@@ -67,15 +68,30 @@
       },
       convertToImg () {
         const reader = new FileReader()
+        const imageFileTypes = ['/png', '/jpg', '/jpeg', '/gif']
         reader.readAsDataURL(this.file.image)
         reader.onload = (e) => {
-          this.previewImage = e.target.result
+          for (let i = 0; i < imageFileTypes.length; i++) {
```

Comment:
Can be simplified (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)
```
if (imageFileTypes.includes(e.target.result)) {
  ...
}
```